repository/scripts/node_modules/widget/overlay.js

```diff
@@ -100,8 +100,25 @@ module.exports = (function () {
 			varbit(player, 18995, tab);
 			break;
 		case 1://Customisations
-			varbit(player, 29607, tab);
-			break;
+		varbit(player, 29607, tab);
+		switch (tab) {
+		case 1://Wardrobe
+		varp(player, 256, 4194274);
```

Comment:
Correct me if I'm wrong, but I'm fairly sure the `varbit(player, 29607, tab)` already does everything in the switch statement). If not, could you figure out the right varbit id to set instead of sending the whole varp? Otherwise it risks breaking other functions.